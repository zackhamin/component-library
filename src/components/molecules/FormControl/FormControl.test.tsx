import { FormControl } from '.';
import { render } from '@testing-library/react';

describe('FormControl', () => {
	it('renders correctly', () => {
		const { container } = render(
			<FormControl id="testId">Children</FormControl>
		);
		expect(container).toMatchSnapshot();
	});
});
