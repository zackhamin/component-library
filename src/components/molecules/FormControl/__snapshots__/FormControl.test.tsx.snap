// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormControl renders correctly 1`] = `
<div>
  <div>
    Children
  </div>
</div>
`;
